### PR TITLE
README.md updates for grpc server going default.

### LINKED ISSUE #790: README.md needs updating for grpc server change

### PR DESCRIPTION
**Pull request by jcvernaleo**
Closes #790

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,23 +32,24 @@ SPV mode that is compatible with dcrd is planned for a future release.
...
+  1. A legacy JSON-RPC server inspired by the Bitcoin Core rpc server
 
      The JSON-RPC server exists to ease the migration of wallet applications
      from Core, but complete compatibility is not guaranteed.  Some portions of
      the API (and especially accounts) have to work differently due to other
      design decisions (mostly due to BIP0044).  However, if you find a
      compatibility issue and feel that it could be reasonably supported, please
-     report an issue.  This server is enabled by default.
+     report an issue.  This server is enabled by default as long as a username
+	 and password are provided.
```
Comment (jrick) [inline README.md]: wrong indentation

File: README.md [hunk 2]
```diff
@@ -32,23 +32,24 @@ SPV mode that is compatible with dcrd is planned for a future release.
...
   2. An experimental gRPC server
 
      The gRPC server uses a new API built for dcrwallet, but the API is not
-     stabilized and the server is feature gated behind a config option
-     (`--experimentalrpclisten`).  If you don't mind applications breaking due
-     to API changes, don't want to deal with issues of the legacy API, or need
-     notifications for changes to the wallet, this is the RPC server to use.
-     The gRPC server is documented [here](./rpc/documentation/README.md).
+     stabilized.  This server is enabled by default and my be disabled with
+	 the config option (`--nogrpc`).  If you don't mind applications breaking
```
Comment (jrick) [inline README.md]: indentation issues here too

File: README.md [hunk 3]
```diff
@@ -32,23 +32,24 @@ SPV mode that is compatible with dcrd is planned for a future release.
...
 
   2. An experimental gRPC server
 
      The gRPC server uses a new API built for dcrwallet, but the API is not
-     stabilized and the server is feature gated behind a config option
-     (`--experimentalrpclisten`).  If you don't mind applications breaking due
-     to API changes, don't want to deal with issues of the legacy API, or need
-     notifications for changes to the wallet, this is the RPC server to use.
-     The gRPC server is documented [here](./rpc/documentation/README.md).
+     stabilized.  This server is enabled by default and my be disabled with
```
Comment (jrick) [inline README.md]: my -> may

File: README.md (hunk 2 above)
Comment (jrick) [inline README.md]: either remove the parentheses around the option or revert "the" back to "a"

File: README.md [hunk 4]
```diff
@@ -32,23 +32,24 @@ SPV mode that is compatible with dcrd is planned for a future release.
...
      The JSON-RPC server exists to ease the migration of wallet applications
      from Core, but complete compatibility is not guaranteed.  Some portions of
      the API (and especially accounts) have to work differently due to other
      design decisions (mostly due to BIP0044).  However, if you find a
      compatibility issue and feel that it could be reasonably supported, please
-     report an issue.  This server is enabled by default.
+     report an issue.  This server is enabled by default as long as a username
+	 and password are provided.
 
   2. An experimental gRPC server
```
Comment (jrick) [inline README.md]: can probably remove the experimental language from here too

Comment (jcvernaleo) [inline README.md]: @jrick Do we still want to say the the 'API is not stabiliized'?

Comment (jrick) [inline README.md]: for now yes but we should begin marking stuff as stabilized.

**Pull request merged**